examples/plot_inverse.py

```diff
@@ -141,3 +142,57 @@
 #
 # .. warning:: To use this pipeline, we need a cortical segmentation of MRI
 #   data, that could be provided by Freesurfer
+
+##############################################################################
+import pickle  # noqa
+from ephypype.gather.gather_results import get_inverse_files  # noqa
+from visbrain.objects import BrainObj, ColorbarObj, SceneObj  # noqa
+
+time_series_files, label_files = get_inverse_files(main_workflow.base_dir,
+                                                   main_workflow.name,
+                                                   subject_ids, session_ids)
+
+time_pts = 30
+
+sc = SceneObj(size=(800, 500), bgcolor=(.1, .1, .1))
+lh_file = op.join(subjects_dir, 'fsaverage', 'label/lh.aparc.annot')
+rh_file = op.join(subjects_dir, 'fsaverage', 'label/rh.aparc.annot')
+for inverse_file, label_file in zip(time_series_files, label_files):
+    # Load files :
+    with open(label_file, 'rb') as f:
+        ar = pickle.load(f)
```

Comment:
doesn't MNE have a reader for labels?

Reply:
yes, but it returns the labels of the "full" source space, it means each label has a lot of vertices, here in the pickle there are the labels of the source space used to compute the inverse